### ISSUE TITLE
Missing session argument in Admin constructor

### ISSUE THREAD

**Issue by fjbosch** [stardog-union/pystardog#121]
When initializing an Admin object, it is sometimes desired to include some custom connection. For the stardog.Connection object that is possible by first creating a requests.Session object and providing it to the constructor of stardog.Connection. However, stardog.Admin accepts similar arguments as Connection, but it does not take a requests.Session as an argument. 

I think Admin should be consistent with Connection and accept a session as an argument. It is also needed for custom connection requirements for both Admin and Connection instances.

[0001-Add-a-request.Session-argument-to-Admin.txt](https://github.com/stardog-union/pystardog/files/9858845/0001-Add-a-request.Session-argument-to-Admin.txt)

**Comment by scardena** [comment]
you are right, we should add a custom Session object to the admin class too. Please feel free to submit a PR. If you have time, writing a test for it will be highly appreciated.
